lib/chef-dk/command/generator_commands.rb

```diff
@@ -69,6 +69,12 @@ module SharedGeneratorOptions
         :description => "Use to set arbitrary attribute KEY to VALUE in the code_generator cookbook",
         :default => [],
         :proc => Proc.new { |s| generator_args <<  s.split("=").map(&:strip) }
+
+      option :branch,
```

Comment:
We're running into an issue where, even though it's only used in generating build cookbooks, the build cookbook recipe is included by the cookbook generator recipe. 

@tyler-ball was helping me try to figure out how we can include that but just in the case of generating a delivery-project backed cookbook, but we were having trouble getting it right. Can you help out today?